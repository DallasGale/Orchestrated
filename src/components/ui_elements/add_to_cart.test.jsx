import React from 'react'
import { render } from 'enzyme'
import toJson from 'enzyme-to-json'
import AddToCart from './add_to_cart'

describe('<AddToCart />', () => {
  test('it renders', () => {
    const Component = render(<AddToCart />)
    expect(toJson(Component)).toMatchSnapshot()
  })
})
