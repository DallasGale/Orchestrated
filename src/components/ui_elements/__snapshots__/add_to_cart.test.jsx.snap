// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddToCart /> it renders 1`] = `
<button
  class="sc-ifAKCX ewMubd"
  type="button"
>
  Add to cart
</button>
`;
